### DISCUSSION TITLE
Fixing the position of Link box in primeng editor

### DISCUSSION THREAD

**Discussion by Sakthikrishna13-10** [primefaces/community#202] [PrimeNG] [answered]
I have implemented primeng editor. I have the issue with positioning of this Link box in primeng editor

For example,
Suppose we have a scenario with lot of contents in prime editor(scollbar is shown)
Once I click on Link toolbar, it is opening the link bar to enter the URL. That works. Refer the image below
(https://github.com/primefaces/community/assets/124769393/63fa72a6-0228-477c-b46c-49038aa2efe5)

Incase if i scroll the content with Link bar open, Link bar goes out of the prime editor. Either Link bar should be within the editor if we scroll the content or it should remove the link bar if we scroll the content. 

Refere the image below
![image](https://github.com/primefaces/community/assets/124769393/2315660c-b28b-489b-ae4b-b3b581386107)

**Comment by cetincakiroglu** [comment]
Hi,

The issue is not clear, could you please share a Stackblitz example or some video recordings about the issue so we can clearly identify it?

**Comment by Sakthikrishna13-10** [comment]
https://github.com/primefaces/community/assets/124769393/ddd7b7c1-73d1-41ba-8f87-bcd4448ce797

please check this video, while scrolling, Link box goes out. Can we fix the position or remove link bar when scrolling

**Reply by cetincakiroglu** [reply, marked as answer]
Hi,

Unfortunately, the issue is beyond PrimeNG, it's about Quill editor and can be reproducible in their own examples https://quilljs.com/playground/#autosave

We cannot provide a solution in this case since another library caused the error. Please open an issue on [Quill Github Repo](https://github.com/quilljs/quill)
